ci/cudf_pandas_scripts/pandas-tests/diff.sh

```diff
@@ -8,14 +8,16 @@
 
 # Hard-coded needs to match the version deduced by rapids-upload-artifacts-dir
 GH_JOB_NAME="pandas-tests-diff / build"
+RAPIDS_FULL_VERSION=$(<./VERSION)
```

Comment:
@raydouglass Instead of introducing a hard-coded version, I'm re-utilizing `VERSION` value that is more than enough for us here thus not needed updating version script entry.